packages/sui-js/test/asyncPipeSpec.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-env mocha */
+import {expect} from 'chai'
+import {asyncPipe} from '../src/pipe/index'
+
+describe('@s-ui/js', () => {
+  const textToUpperCase = text => text.toUpperCase()
```

Comment:
shouldn't you use async functions? what's the point of pipe if this function works with sync function too?

Reply:
I changed it. asyncPipe accept async and sync functions